src/components/hand-tracking-controls.js

```diff
@@ -172,7 +172,7 @@ module.exports.Component = registerComponent('hand-tracking-controls', {
       if (!controller || !this.mesh) { return; }
       this.mesh.visible = false;
       if (!this.hasPoses) { return; }
-      for (var inputjoint of controller.hand.values()) {
+      controller.hand.values().forEach(function (inputjoint) {
```

Comment:
This two are not equivalent. this in the `for` not the same as this in the function scope. In general I favor `for` loops over callbacks that introduce new scopes and are potentially confusing. not a fan of arrow function either since it introduces multiple ways to define a function with different implicit scoping rules. Super confusing specially for newbies.

Reply:
Thanks for catching this!

Have used a classic for loop instead for this one.